musl/arch/riscv64/syscall_arch.h

```diff
@@ -1,7 +1,9 @@
 #define __SYSCALL_LL_E(x) (x)
 #define __SYSCALL_LL_O(x) (x)
 #define UNTRUSTED_MEM_PTR 0x0000001000000000
-#define DEFAULT_MMAP_SIZE 4096 * 8
+#define DEFAULT_BRK_SIZE 4096 * 256
```

Comment:
Why the `DEFAULT_BRK_SIZE` (256 pages) is significantly larger than the size declared in `app.lds`, which is 64 pages?